tools/topology/tests/currency_definitions/snapshot.json

```diff
@@ -182,115 +182,22 @@
-        "network": "NEUTRON",
-        "ch": "channel-209"
-      }
+    "NOLUS": {
+      "NEUTRON": "channel-116",
+      "OSMOSIS": "channel-0"
     },
-    {
-      "a": {
-        "network": "NEUTRON",
-        "ch": "channel-8"
-      },
-      "b": {
-        "network": "AXELAR",
-        "ch": "channel-237"
-      }
-    },
-    {
-      "a": {
-        "network": "NEUTRON",
-        "ch": "channel-1"
-      },
-      "b": {
-        "network": "COSMOS_HUB",
-        "ch": "channel-16"
-      }
+    "OSMOSIS": {
+      "AKASH": "channel-73",
+      "AXELAR": "channel-3",
+      "COSMOS_HUB": "channel-12",
+      "JUNO": "channel-1",
+      "NOLUS": "channel-1636"
     }
-  ],
-  "definitions": [
-    "This is the descriptor of the network topology and currencies of interest for the Nolus AMM protocol.",
-    "",
-    "The network topology is represented as a collection of networks and the ICS-20 transfer channels that connect them.",
-    "Some networks may have DEX services available. Their description contain the pools of interest for the Nolus AMM protocol.",
-    "",
-    "On a given network, each currency is identified by its 'ticker'. A currency with ticker NLS on a network NOLUS is required.",
-    "The Nolus network description should not contain other currencies. They are defined in the 'protocols' object.",
-    "",
-    "A currency is either native or ibc on a network. In the latter case it points to the 'burning' currency residing at a network",
-    "that is at one hop distance. In the former case it is described with name, symbol and number of decimal digits.",
-    "",
-    "The 'name' is a human-readable description of the currency",
-    "",
-    "The 'symbol' is the base denomination of the currency at its native chain.",
-    "",
-    "The 'decimal_digits' value represents the number of decimal digits this denomination has.",
-    "For example, '6' for OSMO means 10^6 units of its base denomination uosmo are equal to 1 OSMO",
-    "",
-    "The 'icon' is an optional currency attribute that provides a visual representation of the currency at that network and the others",
-    "where the currency might be sent over ibc. In other workds, if there is no icon attribute of an ibc originating currency then ",
-    "the icon is derived from the network and currency it points to. This might be applied multiple times traversing the ibc path ",
-    "toward its native network.",
-    "",
-    "The currency symbol at a given network is either equal to the currency 'symbol' if it is a native for that network, or ",
-    "'ibc/' + sha256('transfer' + '/' + ch[0] + '/' + ... + 'transfer' + '/' + ch[k] + '/' + symbol) if it is a native on ",
-    "network 'K' reachable through a sequence of channels with names 'ch[0]', 'ch[1]' ... 'ch[k]' at their minting side.",
-    "More info is available [here](https://github.com/cosmos/ibc-go/blob/c86d27fc280cfb342a9e4689b381e5823441b694/modules/apps/transfer/types/trace.go#L19)."
-  ]
```

Comment:
Have we moved this info to a safe place? If not, please do.
I would suggest a separate Markdown file close to the definitions.

Reply:
disregard, just read thoroughly the PR's description